src/components/analysis-complete-message/analysis-complete-message.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { InfoSprinkle } from 'hadron-react-components';
+import pluralize from 'pluralize';
+
+const SCHEMA_ANALYSIS_DOCS_LINK = 'https://docs.mongodb.com/compass/current/sampling#sampling';
+
+/**
+ * Component for the analysis message.
+ */
+class AnalysisCompleteMessage extends Component {
+  static displayName = 'AnalysisCompleteMessageComponent';
+
+  static propTypes = {
+    sampleSize: PropTypes.number.isRequired
+  }
+
+  _openLink(link) {
+    const { shell } = require('electron');
```

Comment:
Yeah this is done like this a bit of everywhere, I'm not sure why though, it seems an innocent change, let me try :)

Reply:
Even better looks like we have an `openLink` module

Reply:
No .. fake news: the fancy `openLink` is in some other plugin.

I think `shell` is never required "statically" in the whole compass. It should be harmless to do so since is available in both the Main and Renderer process, but makes me think it would probably break some unit tests if we run them outside of karma / electron (not in this case though).

I'd leave this as it is for now, probably we could refactor it everywhere when we have the monorepo and use `openLink` everywhere.